src/language/extension-methods.md

```diff
@@ -220,6 +220,7 @@ extension NumberParsing on String {
 
 The members of an extension can be methods, getters, setters, or operators.
 Extensions can also have static fields and static helper methods.
+Make sure to use the Extension name for invoking them.
```

Comment:
When reading this, it wasn't clear that it was just about the static members. Perhaps we could call that out more directly? Also I believe they can be used without the extension name within the declaration, so we should account for that somehow.

How about something like:

```
To access static members outside of the extension declaration,
invoke them through the declaration name like [class variables and methods][].

[class variables and methods]: /language/classes#class-variables-and-methods
```

This text also references class static members to provide extra context and reading material.

Reply:
Hi, @parlough! 😄 
Yes, it would be better to clarify that an Extension name is only required for static fields and static methods.
Please feel free to co-edit this PR.